app/managers.py

```diff
@@ -0,0 +1,26 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connect = sqlite3.connect("../cinema.sqlite")
+
+    def create(self, first_name: str, last_name: str) -> None:
+        sql = "Insert Into actors (first_name,last_name) VALUES (?,?)"
+        self.connect.execute(sql, (first_name, last_name))
+        self.connect.commit()
+
+    def all(self) -> None:
+        actors = self.connect.execute("Select * From actors")
+        print([actor for actor in actors])
+        return actors
+
+    def update(self, id: int, first_name: str, last_name: str) -> None:
+        sql = "Update actors Set first_name = ?, last_name = ? Where id = ?"
+        self.connect.execute(sql, (first_name, last_name, id))
+        # self.connect.commit()
```

Comment:
uncomment this line

Reply:
Why it's updating, but, for compare, not deleting?